### ISSUE TITLE
Issue vein mining oak trees in the nether with axe (20w18a)

### ISSUE THREAD

**Issue by matthewwyndham** [Elemend/Vein-Miner-Unified#11]
I updated the server to 20w18a and headed to the nether to vein mine my oak trees (to make beds to explode the nether) and my axe completely disappears (tested it with a new diamond axe and it disappeared), and then some stone buttons are placed in my inventory.

I know this datapack isn't tested for 20w18a, but everything works perfectly in the overworld, but in the nether it breaks for some reason. Maybe because the weekly update has changed the blocks...

TLDR;
- version: 20w18a
- In the nether
- Vein Mining connected Oak Logs with diamond axe (full durability)
- Axe disappears
- tree disappears
- no logs in inventory
- stone buttons appear in inventory
- works fine in the overworld but not the nether

Also I tried vein mining netherite with a diamond pick in the nether and I got an inventory full of stone buttons with no pickaxe left

**Comment by Elemend** [comment]
To not give the impression that I wouldn't care...
I'm working on bringing this to the latest snapshot. The reason it was broken is because this datapack isn't intelligent (by any stretch of the imagination) so it just didn't know what to do with the new blocks.
There is also a reason why making this compatible with the latest snapshot is tricky: because Mojang adds a feature or even more new blocks in a later snapshot and something breaks... again... or something can be improved which means redoing some things. So I waited a bit to see what the snapshots bring with them.

**Comment by matthewwyndham** [comment]
Thanks I appreciate all the work you put into this! I thought maybe I could figure out data packs and make a quick fix or a simple alternative, but it’s not easy at all! You are a certified genius and my friends and I appreciate all the work you’ve put into this!!

**Comment by Elemend** [comment]
Alright. I have a Version for 20w21a. There are propably some bugs still in there, but should work mostly.

**Comment by matthewwyndham** [comment]
Thank you so much!
